internal/pkg/usage/usage.go

```diff
@@ -67,6 +67,17 @@ Common Options:
 	-h, --help                      Show this message
 `
 
+var securitySecretStoreSetupUsageStr = `
+Usage: %s [options]
+Server Options:
+	--insureskipverify=true/false			Indicates if skipping the server side SSL cert verifcation, similar to -k of curl
```

Comment:
would prefer to just call this insecure, keeping in line with curl

Reply:
This option is read into `var ensureSkipVerify bool`, so maybe insecure was not the intention here.  @tingyuz ?

Reply:
Nevermind... I think it's just typos both ways.

Reply:
this should be "insecureXXX", will change it.